resources/public/api/conf/1.0/scenarios/retrieve_charitable_givings.md

```diff
@@ -0,0 +1,19 @@
+<p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+<table>
+    <thead>
+        <tr>
+            <th>Header Value (Gov-Test-Scenario)</th>
+            <th>Scenario</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><p>N/A - DEFAULT</p></td>
+            <td><p>Simulate a successful response.</p></td>
```

Comment:
Can you change `Simulate` to `Simulates` please

Reply:
Done